src/service/extensions-impl.js

```diff
@@ -238,15 +241,19 @@ export class Extensions {
    * Returns the promise that will be resolved when the extension has been
    * loaded. If necessary, adds the extension script to the page.
    * @param {string} extensionId
+   * @param {string=} extensionVer
    * @param {boolean=} stubElement
    * @return {!Promise<!ExtensionDef>}
    */
-  loadExtension(extensionId, stubElement = true) {
+  loadExtension(extensionId,
+      extensionVer = DefaultExtensionVersion, stubElement = true) {
```

Comment:
I am expecting more strict check than `parseFloat` though. For example `loadExtension('amp-ad', '1.0')` should not be valid since amp-ad doesn't have a 1.0 version.
Or `loadExtension('amp-sticky-ad', '0,1')` and `loadExtension('amp-sticky-ad', '1.0')` should not be called in same page.